custom_components/places/update_sensor.py

```
@@ -543,47 +647,81 @@ async def process_osm_update(self, now: datetime) -> None:
+                        changed_at = changed_at.replace(tzinfo=now.tzinfo or UTC)
+                    elif now.tzinfo is not None:
+                        changed_at = changed_at.astimezone(now.tzinfo)
+            if await self.get_seconds_from_last_change(now) >= 86399:
+                self.coordinator.set_native_value(state)
+                await self.change_show_time_to_date()
+                return
+            suffix = f" (since {changed_at.hour:02}:{changed_at.minute:02})"
+            state = f"{state[: 255 - len(suffix)]}{suffix}"
+        else:
+            state = state[:255]
+        self.coordinator.set_attr(ATTR_SHOW_DATE, False)
```

Ruff (0.16.1)

[warning] Boolean positional value in function call

(FBT003)